src/Tests/dotnet-new.Tests/Approvals/DotnetNewHelpTests.CanShowHelp_Create_common.verified.txt

```diff
@@ -0,0 +1,18 @@
+﻿Description:
+  Instantiates a template with given short name. Same as 'dotnet new <template name>'.
```

Comment:
@baronfel please check the wording here.

Reply:
I'd suggest:

> An alias of 'dotnet new &lt;template name&gt;'